<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setpassword</class>
 <widget class="QDialog" name="setpassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>密码设置</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>773</x>
     <y>130</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>345</width>
     <height>45</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title_2">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>10</y>
     <width>314</width>
     <height>45</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密   码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttontemp3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>708</y>
     <width>120</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>773</x>
     <y>203</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>846</x>
     <y>276</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttontemp2">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>708</y>
     <width>120</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="buttontemp5">
   <property name="geometry">
    <rect>
     <x>698</x>
     <y>708</y>
     <width>120</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_0">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>349</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttontemp4">
   <property name="geometry">
    <rect>
     <x>534</x>
     <y>708</y>
     <width>120</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelbuttonarea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>703</y>
     <width>1004</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelmessage">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>608</y>
     <width>721</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>输入以上各项，按[设置]键，结束本次设置。</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>773</x>
     <y>276</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>846</x>
     <y>130</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>276</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>846</x>
     <y>203</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttontemp1">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>708</y>
     <width>120</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="MyLabel" name="labelpassword_old">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>150</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>100</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>203</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonquit">
   <property name="geometry">
    <rect>
     <x>862</x>
     <y>708</y>
     <width>120</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>退   出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>130</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonrevise">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>430</y>
     <width>143</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>修   订</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_3">
   <property name="geometry">
    <rect>
     <x>669</x>
     <y>10</y>
     <width>345</width>
     <height>45</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>16年06月17日 10:16:20</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonset">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>653</y>
     <width>120</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>设   置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>255</y>
     <width>100</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>新密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>100</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLabel" name="labelpassword_new">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>255</y>
     <width>150</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLabel" name="labelpassword_sure">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>350</y>
     <width>150</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>labelbuttonarea</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>title</zorder>
  <zorder>title_2</zorder>
  <zorder>buttontemp3</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>pushButton_9</zorder>
  <zorder>buttontemp2</zorder>
  <zorder>buttontemp5</zorder>
  <zorder>pushButton_0</zorder>
  <zorder>buttontemp4</zorder>
  <zorder>labelmessage</zorder>
  <zorder>pushButton_8</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_7</zorder>
  <zorder>pushButton_6</zorder>
  <zorder>buttontemp1</zorder>
  <zorder>labelpassword_old</zorder>
  <zorder>label</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>buttonquit</zorder>
  <zorder>pushButton_1</zorder>
  <zorder>buttonrevise</zorder>
  <zorder>title_3</zorder>
  <zorder>buttonset</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>labelpassword_new</zorder>
  <zorder>labelpassword_sure</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header>mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
